src/pool/srv_pool.c

```diff
@@ -1698,6 +1700,7 @@ pool_svc_map_dist_cb(struct ds_rsvc *rsvc)
 		D_ERROR(DF_UUID": failed to distribute pool map %u: %d\n",
 			DP_UUID(svc->ps_uuid), map_version, rc);
 
+	svc->ps_global_map_version = max(svc->ps_global_map_version, map_version);
```

Comment:
[Requesting changes] We should not update `ps_global_map_version` when `rc != 0`, should we?

```
if (rc != 0) {
    D_ERROR(DF_UUID": failed to distribute pool map %u: %d\n",
            DP_UUID(svc->ps_uuid), map_version, rc);
    goto out;
}

svc->ps_global_map_version = max(svc->ps_global_map_version, map_version);
```

Reply:
ah, right. Thanks